### PR TITLE
Adopt dynamic component service

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: src/app/public/modules/toast/toast-adapter.service.spec.ts [hunk 1]
```diff
@@ -41,16 +41,16 @@ describe('Toast adapter service', () => {
...
     });
-    adapter = TestBed.get(SkyToastAdapterService);
+    // adapter = TestBed.get(SkyToastAdapterService);
   });
 
-  it('should append element to body', () => {
-    adapter.appendToBody(undefined);
-    expect(rendererCallCounts.appendCalledCount).toBe(1);
-  });
+  // it('should append element to body', () => {
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/toast/toast-adapter.service.spec.ts]: ?

Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/toast/toast-adapter.service.spec.ts]: These need removed. I forgot to go back and do that apparently.

File: package.json [hunk 2]
```diff
@@ -29,14 +29,14 @@
...
-    "@skyux/core": "^3.1.0",
+    "@skyux/core": "^3.5.0",
     "@skyux/i18n": "^3.3.0",
     "@skyux/indicators": "^3.0.0-rc.1"
   },
   "dependencies": {},
   "devDependencies": {
-    "@blackbaud/skyux": "2.28.1",
-    "@blackbaud/skyux-builder": "1.28.0",
+    "@blackbaud/skyux": "2.33.1",
```
Comment (Blackbaud-AlexKingman) [inline package.json]: I think we're up to `2.34` now.

Comment (Blackbaud-TrevorBurch) [reply package.json]: Yup. I bumped it :)

File: src/app/public/modules/toast/toast-adapter.service.spec.ts (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/toast/toast-adapter.service.spec.ts]: I edited that file to now test the one untested function and removed the tests for the removed methods.

**Comment by Blackbaud-TrevorBurch** [comment]
FYI - the baseline screenshots were wrong when I inspected them. I have removed them in order to correct this so that my tests pass.

File: src/app/public/modules/toast/toast-adapter.service.spec.ts [hunk 3]
```diff
@@ -1,10 +1,10 @@
 // #region imports
 import {
-  RendererFactory2
+  RendererFactory2, ApplicationRef
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/toast/toast-adapter.service.spec.ts]: Multiline.

File: src/app/public/modules/toast/toast-adapter.service.spec.ts [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
 } from '@angular/core';
 
 import {
-  TestBed
+  TestBed, inject
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/toast/toast-adapter.service.spec.ts]: Multiline.

File: src/app/public/modules/toast/toast.service.spec.ts [hunk 5]
```diff
@@ -4,7 +4,7 @@ import {
 } from '@angular/core';
 
 import {
-  TestBed
+  TestBed, inject
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/toast/toast.service.spec.ts]: Multiline.

**Review by Blackbaud-SteveBrush — COMMENTED** [review]
A few minor things.

File: src/app/public/modules/toast/toast-adapter.service.spec.ts (hunk 3 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/toast/toast-adapter.service.spec.ts]: Done

File: src/app/public/modules/toast/toast.service.spec.ts (hunk 5 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/toast/toast.service.spec.ts]: Done

File: src/app/public/modules/toast/toast-adapter.service.spec.ts (hunk 4 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/toast/toast-adapter.service.spec.ts]: Done

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**